Public/Sdk/SelfHost/BuildXL/Tools/QTest/Tool.QTestRunner.dsc

```diff
@@ -180,12 +180,9 @@ export function runQTest(args: QTestArguments): Result {
             qTestContextInfoPath,
         ],
         untrackedScopes: [
-            d`d:/data`,
-            d`d:/app`,

```

Comment:
Friendly reminder to not merge this until the GBR deployment that defaults to the session global untracking behavior

Reply:
Sure, this is a draft PR. I won't make it ready until GBR is ready.

Reply:
The GBR has been deployed, I am going to merge this PR.